src/applications/personalization/profile-2/sass/account-security.scss

```diff
@@ -0,0 +1,7 @@
+.account-security-content {
+  .form-expanding-group-open {
+    ul {
+      list-style-type: disc;
```

Comment:
since this is the first time I'm seeing us use this bullet style we should confirm with @tressaellen we don't want the standard square bullet we use elsewhere.

Reply:
@erikphansen @sanlouise - Let's stick with the standard square bullets. I missed that in the style guide and I see no reason to change it to a disc.